utils/not_doctested.txt

```diff
@@ -431,7 +431,6 @@ src/transformers/models/blip/modeling_blip_text.py
 src/transformers/models/blip/modeling_tf_blip_text.py
 src/transformers/models/blip_2/configuration_blip_2.py
 src/transformers/models/blip_2/convert_blip_2_original_to_pytorch.py
-src/transformers/models/blip_2/modeling_blip_2.py
```

Comment:
@NielsRogge  enabled doctest for this file (but in the wrong file). Here is just to keep up with the change in #24854